docs/topic/infrastructure/cluster-design.md

```diff
@@ -89,6 +91,28 @@ that `prometheus-server` may require more memory than is available.
 On EKS we always use the `r5.xlarge` nodes to avoid running low on allocatable
 pods.
 
+#### For notebook node pool
```

Comment:
While I totally agree with you regarding this language in user-facing docs, these are engineering facing docs referring to node pools, and the node pool is literally called "notebook" right now.

So we should either have language that is consistent with our code (i.e., revert back to saying "notebook node pool") or update our code to deploy node pools called "user-server" (which will be a lot of work, but doesn't mean we shouldn't do it necessarily). Third option: keep the heading as "user server node pool" and add a callout explaining that in our infrastructure, these are called "notebook node pools".

https://github.com/2i2c-org/infrastructure/blob/d4224ce65d53ee29656bef6d45cbf7f3d0d10df8/terraform/gcp/cluster.tf#L243

Reply:
I am trying to avoid a situation where an engineer is confused by looking for a node pool in a cloud console or our terraform config called "user server" and it's because it is called "notebook" instead. I wouldn't oppose a proposal where we update our terraform to use "user server" instead. But if we merge docs now, they should be consistent with what exists now since we wouldn't get around to doing that renaming work for a while.

Reply:
I think if we stick to just speaking about notebook node pools, we need to explain we mean that these are user servers anyhow in the end. So going with that straight up is a plus, but if we retain use of notebook somewhere we need to clarify it still in the end no matter what.

To change `resource "google_container_node_pool" "notebook" { ` in terraform, we need to recreate things - right? I'll check... If we need to re-create things in order to rename that, I think we should just stick with that behind the scenes and comment "notebook" refers to "user server".

Reply:
Changing `resource "google_container_node_pool" "notebook" { ` prompted a recreation of all nodes, while changing variable names didn't - but caused some node version upgrades.

I'm open to settling for anything, but I don't think we can find capacity to re-create all node pools for a naming update in terraform config. With that in mind, I lean towards comrpomises where we still steer towards speaking about "user servers" primarily, and adding callouts or comments about its mentioned as "notebook nodes" behind the scenes sometimes.

Reply:
@sgibson91, @consideRatio, I just added a new commit, that tries to clarify naming of the actual k8s resource vs what they mean as a concept. Hope this solves both your concerns. LTMK what you think.